firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestoreSettings.java

```diff
@@ -145,6 +145,30 @@ public Builder setCacheSizeBytes(long value) {
       return this;
     }
 
+    /** @return the host of the Cloud Firestore backend. */
+    @Nullable
+    public String getHost() {
+      return host;
+    }
+
+    /** @return boolean indicating whether SSL is enabled or not. The default is to use SSL. */
```

Comment:
I'm sorry for the unnecessary back-and-forth, I missed my early cup of copy and it shows. As there's always a valid value returned, we can get rid of the comment with the default value. I've suggested the changes to simplify this.

Reply:
@rlazo No problems! :) All done.